### PR TITLE
feat: dwh stats handle

### PR DESCRIPTION
**Pull request by oopcode**
The new `GetStats()` handle returns current DWH stats.

### CODE REVIEW

File: insonmnia/dwh/sql.go [hunk 1]
```diff
@@ -1134,6 +1134,61 @@ func (m *sqlStorage) GetLastEvent(conn queryConn) (*blockchain.Event, error) {
 	}, nil
 }
 
+func (m *sqlStorage) getStats(conn queryConn) (*sonm.DWHStatsReply, error) {
+	var (
+		numCurrDealsQ, _, _ = m.builder().Select("count(Id)").From("Deals").Where("Status=1").Prefix("(").Suffix(")").ToSql()
```
Comment (sshaman1101) [inline insonmnia/dwh/sql.go]: maybe pass const with status via `.Where`'s  args?

Comment (oopcode) [reply insonmnia/dwh/sql.go]: not sure how this is any better, do you have any ideas?

Comment (oopcode) [reply insonmnia/dwh/sql.go]: ok, sounds reasonable

File: insonmnia/dwh/sql.go [hunk 2]
```diff
@@ -1134,6 +1134,61 @@ func (m *sqlStorage) GetLastEvent(conn queryConn) (*blockchain.Event, error) {
...
+		numDealsQ, _, _     = m.builder().Select("count(Id)").From("Deals").Prefix("(").Suffix(")").ToSql()
```
Comment (avsigaev) [inline insonmnia/dwh/sql.go]: add filter: 'where status = 1'.

We need number of current active deals here

File: insonmnia/dwh/server.go [hunk 3]
```diff
@@ -399,3 +425,10 @@ func (m *DWH) GetWorkers(ctx context.Context, request *sonm.WorkersRequest) (*so
 
 	return &sonm.WorkersReply{Workers: workers, Count: count}, nil
 }
+
+func (m *DWH) GetStats(ctx context.Context, request *sonm.Empty) (*sonm.DWHStatsReply, error) {
+	m.mu.RLock()
+	defer m.mu.RUnlock()
+
+	return m.stats, nil
```
Comment (antmat) [inline insonmnia/dwh/server.go]: this is returned by pointer and can be serialized during modification

**Pull request merged**